packages/devtools/lib/src/timeline/frames_bar_chart.dart

```diff
@@ -40,7 +40,7 @@ class FramesBarChart extends CoreElement with SetStateMixin {
     });
   }
 
-  static const int chartHeight = 160;
+  static const int chartHeight = 230;
```

Comment:
hmm. this is a significant increase. @kenzieschmoll does making the height be this high work for you or do we need to just have the views use different heights?

Reply:
I'm not crazy about increasing the height of the FPS chart. This further reduces the space we have for the flame chart (which may become significantly taller soon if we can add cpu sampling to the flame chart). The FPS chart has the least granularity on the page, so it doesn't need a ton of space to get its message across. The flame chart (and maybe details section if we keep it), however, need to show a lot of data at once, so I think we should prioritize the real estate on the page for this (these) sections.

 Can we pass chart height in as a parameter? That way we can have different sizes for each page.